host/xtest/pkcs11_1000.c

```diff
@@ -8444,6 +8478,12 @@ static CK_RV test_rsa_aes_wrap(ADBG_Case_t *c, CK_SESSION_HANDLE session,
 		  t->target.rsa.prime1_len },
 		{ CKA_PRIME_2, (CK_VOID_PTR)t->target.rsa.prime2,
 		  t->target.rsa.prime2_len },
+		{ CKA_EXPONENT_1, (CK_VOID_PTR)t->key.exp1,
+		  t->key.prime2_len },
+		{ CKA_EXPONENT_2, (CK_VOID_PTR)t->key.exp2,
+		  t->key.prime2_len },
+		{ CKA_COEFFICIENT, (CK_VOID_PTR)t->key.coeff,
+		  t->key.coeff_len },
```

Comment:
This info used here (`t->key.*`) belong to the wrapping key, not the to target wrapped/unwrapped key which `target_key_template` relates to.